views/index.ejs

```diff
@@ -1,13 +1,26 @@
 <html>
 <head>
 	<title>ACM @ UNO</title>
+   <link rel="stylesheet" href="http://s3.amazonaws.com/codecademy-content/courses/ltp/css/bootstrap.css">
```

Comment:
It would probably look better if we didn't use a resource hosted by `code academy`
Bootstrap is a very popular resource and there are many CDN's out there that host the code